mobile/lib/pages/single_recording_page.dart

```diff
@@ -0,0 +1,151 @@
+import 'package:audio_service/audio_service.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:radioaktywne/components/ra_playbutton.dart';
+import 'package:radioaktywne/components/ra_player/ra_player_recources.dart';
+import 'package:radioaktywne/components/utility/custom_padding_html_widget.dart';
+import 'package:radioaktywne/components/utility/ra_progress_indicator.dart';
+import 'package:radioaktywne/extensions/extensions.dart';
+import 'package:radioaktywne/models/recording_info.dart';
+import 'package:radioaktywne/resources/fetch_data.dart';
+import 'package:radioaktywne/resources/ra_links.dart';
+import 'package:radioaktywne/resources/ra_page_constraints.dart';
+import 'package:radioaktywne/state/audio_handler_cubit.dart';
+
+/// Page displaying a single recording.
+class SingleRecordingPage extends StatefulWidget {
+  const SingleRecordingPage({super.key, required this.recording});
+  final RecordingInfo recording;
+
+  @override
+  _SingleRecordingPageState createState() => _SingleRecordingPageState();
+}
+
+class _SingleRecordingPageState extends State<SingleRecordingPage> {
+  String _fullImagePath = '';
+
+  /// Space between things on the page.
+  static const SizedBox _emptySpace = SizedBox(height: 9);
```

Comment:
This also looks like something that could be defined in RaPageConstraints somewhere, although I'm not so sure on this one, so please check that. Generally speaking, the app will feel much more consistent / cohesive if we use same padding in similar places all across the entire app - that's why I feel a universal padding for a gap like this should be already defined in RaPageConstraints.